corehq/apps/export/views/incremental.py

```diff
@@ -23,55 +22,123 @@
         return context
 
-    def get_success_url(self):
-        # On save, stay on the same page
-        return reverse(self.urlname, kwargs={'domain': self.domain})
+    @property
+    def paginated_list(self):
+        for incremental_export in self.base_query.all():
+            yield {
+                "itemData": self._item_data(incremental_export),
+                "template": "base-incremental-export-template",
+            }
+
+    def _item_data(self, incremental_export):
+        return {
+            "id": incremental_export.id,
+            "active": incremental_export.active,
+            "name": incremental_export.name,
+            "export_instance": incremental_export.export_instance.name,
+            "connection_settings": incremental_export.connection_settings.name,
+            "updateForm": self.get_update_form_response(
+                self.get_update_form(instance=incremental_export)
+            ),
+        }
+
+    def post(self, *args, **kwargs):
+        return self.paginate_crud_response
+
+    def get_deleted_item_data(self, item_id):
+        deleted_export = IncrementalExport.objects.get(id=item_id, domain=self.domain)
+        deleted_export.delete()
+        return {
+            "itemData": self._item_data(deleted_export),
+            "template": "deleted-incremental-export-template",
+        }
+
+    def get_create_form(self, is_blank=False):
+        if self.request.method == "POST" and not is_blank:
+            return IncrementalExportForm(self.request, self.request.POST)
+        return IncrementalExportForm(self.request)
+
+    create_item_form_class = "form form-horizontal"
+
+    def get_create_item_data(self, create_form):
+        new_item = create_form.save()
+        return {
+            "itemData": self._item_data(new_item),
+            "template": "base-incremental-export-template",
+        }
+
+    def get_update_form(self, instance=None):
+        if instance is None:
+            instance = IncrementalExport.objects.get(
+                id=self.request.POST.get("id"), domain=self.domain
+            )
+        if self.request.method == "POST" and self.action == "update":
+            return UpdateIncrementalExportForm(self.request, self.request.POST, instance=instance)
+        return UpdateIncrementalExportForm(self.request, instance=instance)
 
-    def form_valid(self, form):
-        self.object = form.save()  # Saves the forms in the formset
-        return super().form_valid(form)
+    def get_updated_item_data(self, update_form):
+        item = update_form.save()
+        return {
+            "itemData": self._item_data(item),
+            "template": "base-incremental-export-template",
+        }
 
 
 class IncrementalExportLogView(GenericTabularReport, DatespanMixin):
-    name = 'Incremental Export Logs'
-    base_template = 'reports/base_template.html'
-    section_name = 'Project Settings'
-    slug = 'incremental_export_logs'
+    name = "Incremental Export Logs"
```

Comment:
(Curious: You have feelings about quotation marks? Choosing double over single is one of the two more contentious issues about Black (the other being its 88-character line length). Do you have feelings about auto formatters? Do you use one?)

Reply:
I don't. I actually ran black on that file, but probably shouldn't.